gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/validation/ValidationJob.java

```diff
@@ -301,7 +340,8 @@ private void processPartitionedTable(ConvertibleHiveDataset hiveDataset, AutoRet
     Closer closer = Closer.create();
 
     try {
-      statement = this.hiveJdbcConnector.getConnection().createStatement();
+      HiveJdbcConnector hiveJdbcConnector = HiveJdbcConnector.newConnectorWithProps(props);
```

Comment:
Will that create many connections than necessary? Also, where is the connection closed?